modules/twinklewarn.js

```diff
@@ -1081,25 +1086,31 @@ Twinkle.warn.callback.change_category = function twinklewarnCallbackChangeCatego
 		$('select[name=sub_group]').chosen('destroy');
 	}
 
-	var value = e.target.value;
+	var value = e.target.value; // Gets destroyed in the kitchensink loop
```

Comment:
```suggestion
	var value = e.target.value;
```
no longer happening.